app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -180,6 +180,10 @@ def validate_ip_address(_field, _values, _dlg, _fld, value)
     return _('IP-address field has to be either blank or a valid IPv4 address') unless valid
   end
 
+  def validate_placement_group(_filed, _value, _dlg, _fld, value)
```

Comment:
Minor but the rest of these method args have `_values` not `_value`: for example `(_field, _values, _dlg, _fld, value)`
It is confusing to have `_value` (which means we're passed `value` but it isn't used) then also `value` later on.

Reply:
Unresolving this comment since it is still applicable